packages/shared/src/globalsWhitelist.ts

```diff
@@ -1,7 +1,7 @@
 export const globalsWhitelist = new Set(
-  (
-    'Infinity,undefined,NaN,isFinite,isNaN,parseFloat,parseInt,decodeURI,' +
-    'decodeURIComponent,encodeURI,encodeURIComponent,Math,Number,Date,Array,' +
-    'Object,Boolean,String,RegExp,Map,Set,JSON,Intl'
-  ).split(',')
+  [
+    ...['Infinity', 'undefined', 'NaN', 'isFinite', 'isNaN', 'parseFloat', 'parseInt', 'decodeURI'],
+    ...['decodeURIComponent', 'encodeURI', 'encodeURIComponent', 'Math', 'Number', 'Date', 'Array'],
+    ...['Object', 'Boolean', 'String', 'RegExp', 'Map', 'Set', 'JSON', 'Intl']
+  ]
```

Comment:
What's the point of spreading several array literals into a new array, instead of just creating a single array literal?

Reply:
I originally wanted to merge into an array, but I was not sure about the previous intention, so I used a conservative approach. 
Now I will merge it into one.